### PR TITLE
Add NGINX Cerbot Dependency | Add sudo commands | Add SSH Keys

### PR DESCRIPTION
**Pull request by ncorder**
- [ ] Add NGINX Cerbot Dependency 
- [ ] Add sudo commands
- [ ] Add SSH Keys
- [ ] Disable Sudo Password
- [ ] Create SSH Keys

### CODE REVIEW

**Comment by ncorder** [comment]
Let me know if you think anything should be phrased differently as well.

File: README.md [hunk 1]
```diff
@@ -9,7 +9,45 @@ To run this ansible playbook, you need to:
 - Have a server / VPS where lemmy will run.
 - Configure a DNS `A` Record to point at your server's IP address.
 - Make sure you can ssh to it: `ssh <your-user>@<your-domain>`
-- Install [Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) on your local machine.
+- Install [Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) on your local machine.<br><br>
+Can be ingnored if you are using another web server:
+- Install [NGINX](http://nginx.org/en/download.html) | Linux Install Instruction: [https://nginx.org/en/linux_packages.html](https://nginx.org/en/linux_packages.html)
+- Install the Python NGINX Plugin for Certbot [`python3-certbot-nginx`](https://packages.debian.org/bullseye/python3-certbot-nginx)
```
Comment (dessalines) [inline README.md:15]: These aren't necessary, ansible installs these automatically if they aren't there.

File: README.md [hunk 2]
```diff
@@ -9,7 +9,45 @@ To run this ansible playbook, you need to:
 - Have a server / VPS where lemmy will run.
 - Configure a DNS `A` Record to point at your server's IP address.
 - Make sure you can ssh to it: `ssh <your-user>@<your-domain>`
-- Install [Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) on your local machine.
+- Install [Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) on your local machine.<br><br>
```
Comment (dessalines) [inline README.md:12]: Why are there <br>'s here? That's not markdown.

File: README.md [hunk 3]
```diff
@@ -9,7 +9,45 @@ To run this ansible playbook, you need to:
...
+Can be ingnored if you are using another web server:
+- Install [NGINX](http://nginx.org/en/download.html) | Linux Install Instruction: [https://nginx.org/en/linux_packages.html](https://nginx.org/en/linux_packages.html)
+- Install the Python NGINX Plugin for Certbot [`python3-certbot-nginx`](https://packages.debian.org/bullseye/python3-certbot-nginx)
+
+You must also have a user, with home access.
+```
+adduser --home /home/lemmy lemmy
+```
```
Comment (dessalines) [inline README.md:20]: This is going to be different for different distros, so shouldn't be here.

File: README.md [hunk 4]
```diff
@@ -9,7 +9,45 @@ To run this ansible playbook, you need to:
...
+Lemmy's ansible currently requires no password with sudo.
+run `visudo` and append the following:
```
Comment (dessalines) [inline README.md:22]: Any ansible that runs docker is going to need root. 

Everything below is very basic linux server admin. I don't want to maintain something that's easily searchable for your distro.

**Comment by dessalines** [comment]
In the future, pull the repo down locally instead of making 10+ edits on github.

File: README.md (hunk 1 above)
Comment (ncorder) [reply README.md:15]: Ansible did not install them automatically on mine but that might have been a local issue

File: README.md (hunk 2 above)
Comment (ncorder) [reply README.md:12]: `<br>` is markdown compliant

File: README.md (hunk 3 above)
Comment (ncorder) [reply README.md:20]: I believe `adduser` is supported nearly everywhere.

It is in old Slackware versions.

**Pull request closed without merging**